### PR TITLE
Complete signup page UI changes

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look.

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: web/react/components/signup_user_complete.jsx [hunk 1]
```diff
@@ -117,24 +117,23 @@ module.exports = React.createClass({
...
         return (
             <div>
                 <img className="signup-team-logo" src="/static/images/logo.png" />
-                <h4>Welcome to { config.SiteName }</h4>
+                <h3 className="text-center">Signup to { config.SiteName }</h3>
                 <div className="form-group form-group--small">
                     <span></span>
                 </div>
                 { signup_message }
-                <p>Your username can be made of lowercase letters and numbers.</p>
```
Comment (rgarmsen2295) [inline web/react/components/signup_user_complete.jsx]: I noticed that this is repeated in the oauth user signup page but if it is removed here will users who are completing the normal sign-up flow see this anywhere else?  I just remember this previously being an issue with signup flow due to users not realizing that they couldn't use uppercase letters until an error message appeared which isn't good.

Comment (asaadmahmoodspin) [inline web/react/components/signup_user_complete.jsx]: Yeah I think you're right, lets add it back in.

**Comment by asaadmahmoodspin** [comment]
Added it back in but below username.

**Comment by rgarmsen2295** [comment]
Yup I see it, +1 from me, looks good

**Pull request merged**